bin/tart2ms

```diff
@@ -38,7 +38,7 @@ if __name__ == "__main__":
     
     '''
     New CLI:    --rephase <target> Perform rephasing of _all_ input visibility snapshots (single integration visibility fields) to the specified phase center.
-                                   This does not change the number of fields that will be output. There will be one field with name J_XXXXXX per visibility integration.
+                                   This does not change the number of scans that will be output. There will be one field with name J_XXXXXX.
```

Comment:
Not sure what a scan is?. Need to document terminology here... https://tart.elec.ac.nz/tart_website/docs/tutorial-basics/terminology. 

Changes go here: https://github.com/tart-telescope/tart_website/blob/main/docs/tutorial-basics/terminology.md

Reply:
Currently we define scan boundaries as integration boundaries (it differs from telescope to telescope). Typically scan means a contigeously tracked object or undertaking (e.g. target field / calibrator field / slewing scan etc.). We can define it differently when fringe stopping and set it to a larger interval which is why I'm hesitant to firmly define the scan boundary. Do you want to fix the definition to mean integration boundary?

Reply:
We need to define it clearly.  And tighten up terminology.  We have used "single integration visibility fields" which should be tightened up. 

 How about: 

- A snapshot is a single-integration set of visibility measurements, one from each pair of antennas
- A scan is a continuous sequence of one or more snapshots. Used to logically group shapshots for processing
- A field is the location of the phase-center of a snapshot. By default it is the local zenith,  expressed in celestial coordinates.

Then

- --rephase <target>  : Changes the phase center of a set of snapshots, but doesn't combine them.
- --single-field   : Combines the snapshots into a single scan with a common phase center.